app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -14,6 +19,44 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
   require_nested :Template
   require_nested :Vm
 
+  # The list of absolute URI paths where the API can be available:
+  CANDIDATE_API_PATHS = [
+    '/ovirt-engine/api',
+    '/api',
+  ].freeze
+
+  # Checks if the API is available in the given candidate path. It does so sending a request without authentication. If
+  # the API is available there it will respond with the "WWW-Autenticate" header and with the "RESTAPI" or "ENGINE"
+  # realm.
+  def probe_api_path(server, port, path)
```

Comment:
I expected this in the `ovirt` gem itself so the caller (i.e. the ManageIQ provider) shouldn't have to know the details.